### ISSUE TITLE
Tokyo Tyrant (Tokyo Cabinet server)

### ISSUE THREAD

**Issue by lumina7** [johnjansen/crystal-libraries-needed#115]
Tokyo Tyrant (tokyo cabinet server bindings).  Used by cloudfare/mixi.

https://fallabs.com/tokyotyrant/

Ruby bindings:
https://fallabs.com/tokyotyrant/rubypkg/

Go bindings:
https://github.com/patrickxb/gotyrant

Thank you.

**Comment by Sija** [comment]
fwiw, Tokyo Tyrant hasn't been maintained for a looooong time already...

**Comment by lumina7** [comment]
https://news.ycombinator.com/item?id=2526355

**Comment by Sija** [comment]
@lumina7 this article is from 2011 if you haven't noticed...

**Comment by watzon** [comment]
Yeah even the Go bindings, and probably the Ruby bindings, haven't been updated in 10 years
